src/dune_pkg/ocamlformat.mli

```diff
@@ -0,0 +1,3 @@
+val pkg_name : string
```

Comment:
This module needs some docs. As mentioned earlier in this PR, there are better alternatives to strings for these values.

Reply:
Done. All that's left undocumented about this module is the binary name which I will remove soon.